lib/engine.py

```diff
@@ -541,22 +538,19 @@ def format_insert_value(self, value, datatype):
                 return "null"
         elif datatype in ("double", "decimal"):
             if strvalue:
-                return strvalue
+                decimals = float(strvalue)/1.0
```

Comment:
Are you sure we need the `/1.0`. I understand that we need `float` to convert `12` into `12.0`, but I don't see any difference in behavior between `float(strvalue)` and `float(strvalue)/1.0`.